packages/components/src/components/post-language-option/post-language-option.tsx

```diff
@@ -56,6 +56,11 @@ export class PostLanguageOption {
    */
   @Prop() variant?: SwitchVariant | null;
 
+  /**
+   * The type of the post-language-switch parent (dynamically set by the parent)
+   */
```

Comment:
Can you make the description more specific so that we can easily differentiate the `variant` prop form the `type` prop?